packages/web-components/src/components/va-combo-box/test/va-combo-box.e2e.ts

```diff
@@ -352,7 +352,7 @@ describe('va-combo-box', () => {
     expect(firstComboValue).toBe('foo');
   });
 
-  it('handles search result status aria', async () => {
+  it.only('handles search result status aria', async () => {
```

Comment:
The `.only` on this test should be removed.

Reply:
@jamigibbs - done!